frontend/src/app/Settings/FacilityForm.test.tsx

```diff
@@ -213,6 +215,46 @@ describe("FacilityForm", () => {
     const state = await screen.findByText("Palau", { exact: false });
     expect(state).toBeInTheDocument();
   });
+
+  describe("CLIA number validation", () => {
+    describe("when validation is not required for state", () => {
```

Comment:
Should there be a sibling `describe` that tests when the CLIA number is required by the state, testing both correctly-formatted CLIA numbers and incorrectly-formatted CLIA numbers?

Reply:
It wouldn't hurt. The happy-path `submits a valid form` test in this file implicitly covers the typical validation-required case but I've added another to verify the behavior on incorrectly formatted CLIA numbers.

See 429bdd6.